src/actions/articleActions.js

```diff
@@ -15,10 +15,10 @@ export function updateArticle(payload) {
   };
 }
 
-export function fetchArticlesByProductId(id) {
+export function fetchArticlesByProductId(id, sortKey) {
+  const query = (!Number.isNaN(id) ? "?product=" + id : "") + (sortKey ? "&sort=" + sortKey : "")
```

Comment:
Burde vi heller ha to metoda? Sånn som eg opptatt den metoden her, så må vi alltid ha en "productID" for å motta sorterte artikla.

Reply:
fixed 3049a5cd1350188204d52871c485232422dcb150